src/components/OptionsSelector.js

```diff
@@ -213,47 +225,101 @@ class OptionsSelector extends Component {
         this.props.onSelectRow(option);
     }
 
+    /**
+     * @param {Number} focusedIndex
+     */
+    selectFocusedIndex(focusedIndex) {
+        if (!this.state.allOptions[focusedIndex]) {
+            return;
+        }
+
+        this.selectRow(this.state.allOptions[focusedIndex]);
+
+        if (!this.props.canSelectMultipleOptions) {
+            return;
+        }
+
+        this.scrollToIndex(0);
```

Comment:
@roryabraham I was expecting that we scroll to the first row **only if the search query** has changed. 

**Observation**: On selecting a row using `Enter` key, we're scrolling to the first index. (Note that the search query hasn't changed). 

**Problem**: This may not be ideal if we wanna select users that are grouped together at the bottom of the list, because we'll have to keep navigating the list every time a user is selected.

**Solution**: Don't scroll to the first index when a row is selected using `Enter` key.

- returning early by diffing `prevProps.value` in `ComponentDidUpdate()`
- Remove `this.scrollToIndex(0)` from `selectFocusedIndex()`

https://user-images.githubusercontent.com/29673073/163358693-54de98a4-0dcf-4e2d-9dc7-1bae9d620698.mov

Reply:
Hmmm I actually think that this behavior is what I would expect, per the same reasoning you used above:

> the best result™️ is always at the top.

So if I select a new participant with the Enter key, then scrolling to the first item is good for two reasons:

1. You can easily see that your recent selection is now one of the selected options.
1. You are refocused on the **best result™️** according to the current search query.

> I really appreciate your patience :)

I really appreciate the thorough review and the care you're taking with making our product the best it can be 🙂

Reply:
> This may not be ideal if we wanna select users that are grouped together at the bottom of the list, because we'll have to keep navigating the list every time a user is selected.

I think the mainline use-case would be to enter a search query for the user you're looking for, rather than keying through the whole list.